pmkoalas/models/pnfrag.py

```diff
@@ -331,7 +331,11 @@ def weighted_value_transition(self):
 
 
     def tran_label(self):
-        return self.lookahead.token_str
+        label = self.lookahead.token_str
+        while self.tokens[1].token_info == TokenInfo.LABEL:
```

Comment:
This is confusing in an LL(1) grammar parser, since formally the grammar only requires one token of lookahead. It is also what we were going around in circles on when we last talked about it, so let me try and give an alternative implementation separately.